cfme/automate/explorer/method.py

```diff
@@ -397,22 +421,25 @@ def domain(self):
     def tree_path(self):
         icon_name_map = {'inline': 'fa-ruby', 'playbook': 'vendor-ansible'}
         if self.display_name:
-            return self.parent_obj.tree_path + [
-                (icon_name_map[self.location.lower()], '{} ({})'.format(self.display_name,
-                                                                        self.name))]
+            if self.location in ['Ansible Tower Job Template', 'Ansible Tower Workflow Template']:
```

Comment:
Thoughts on making this strings class level or module level attributes? That way if they need to be updated, you will only have to update them there.

Reply:
Hey John,
- I am not getting what you actually want here. Do you want me to create global list of locations?
- Please explain more and give me example.

Reply:
@ganeshhubale sorry for delayed reply, I must've missed this. Yeah, I was suggesting that you make `global` list lof locations, something like: 

```python,
LOCATIONS =  ['Ansible Tower Job Template', 'Ansible Tower Workflow Template']
```

Reply:
Got it @john-dupuy 
I will update with this change.